Danki2/Assets/Scripts/Camera/CameraShakeManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class CameraShakeManager
+{
+    private CameraShake[] camShakes;
+
+    public CameraShakeManager(int poolSize)
+    {
+        camShakes = new CameraShake[poolSize];
+
+        for (int i = 0; i < poolSize; i++)
+        {
+            camShakes[i] = new CameraShake();
+        }
+    }
+
+    public Vector3 GetShakeVector()
+    {
+        float maxStrength = float.MinValue;
+        CameraShake strongestShake = default;
```

Comment:
Would it be clearer to use null instead of default here and on line 34? To me using default here makes CameraShake appear to be a struct, what do you think?

Reply:
Probably null is better. I default to default because the Linq `.FirstOrDefault()` method is so common.